### PR TITLE
Put logic in functions and added tests

### PR DESCRIPTION
**Pull request by TonyPittella**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 venv
 .DS_Store
```
Comment (dom-edu) [inline .gitignore:2]: maybe add the chromdriver to this as well and remove it from your repo

File: conftest.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+def page_url():
+    req = requests.get(URL)
+    return req
+
+@pytest.fixture
+def page_html(page_url):
+    soup = BeautifulSoup(page_url.content, "html.parser")
+    return soup
+
+SERVER = os.getenv('SERVER')
```
Comment (dom-edu) [inline conftest.py:16]: try to put constants at the top of your program after the imports

File: hacker_news_scrapper.py [hunk 3]
```diff
@@ -25,48 +27,43 @@ def extract_news(url):
...
-    # attr = {'class': 'title', 'valign':' ' })):# 'a': 'storylink'
     for i, tag in enumerate(soup.find_all('td', {'class': 'title', 'valign': ''})):
         cnt += ((str(i+1) + ' :: ' + tag.text + "\n" + '<br>')
                 if tag.text != 'More' else '')
     return cnt
 
-
-CNT = extract_news('https://news.ycombinator.com/')
-print(CNT)
+URL = 'https://news.ycombinator.com/'
```
Comment (dom-edu) [inline hacker_news_scrapper.py:35]: see comment above about constants

File: hacker_news_scrapper.py [hunk 4]
```diff
@@ -25,48 +27,43 @@ def extract_news(url):
...
+CNT = extract_news(URL)
 CONTENT += CNT
 
 CONTENT += ('<br>------<br>')
 CONTENT += ('<br><br> End of message')
 
 print('Composing Email...')
-
-# email credentials
+load_dotenv()
```
Comment (dom-edu) [inline hacker_news_scrapper.py:43]: i don't think you need this line,because you are using `os.getenv()` instead

**Review by dom-edu — CHANGES_REQUESTED** [review]

File: conftest.py (hunk 2 above)
Comment (TonyPittella) [reply conftest.py:16]: will do next commit

File: hacker_news_scrapper.py (hunk 4 above)
Comment (TonyPittella) [reply hacker_news_scrapper.py:43]: i need the load_dotenv() in there, otherwise it throws an error

**Pull request merged**